build/base.mk

```diff
@@ -1,7 +1,9 @@
 UNAME_OS := $(shell uname -s)
 UNAME_ARCH := $(shell uname -m)
 
-CACHE_BASE := $(HOME)/.yarpc-go
+XDG_CACHE_HOME ?= $(HOME)/.cache
+
+CACHE_BASE := $(XDG_CACHE_HOME)/yarpc-go
```

Comment:
Note I rather this be `.cache` always unless `XDG_CACHE_HOME` is set, ie I want this to be in my homedir for mac, and not be in `Library/Caches`. This is so I have easy access to it (other things use `.cache`, I think Docker does for example).